stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/list/adapter/view/internal/MessageReplyView.kt

```diff
@@ -168,18 +174,11 @@ internal class MessageReplyView : FrameLayout {
     }
 
     private fun setAttachmentImage(message: Message) {
-        val attachment = message.attachments.lastOrNull()
-        if (attachment == null) {
-            binding.logoContainer.isVisible = false
+        if (ChatUI.quotedAttachmentFactoryManager.canHandle(message)) {
+            binding.attachmentContainer.isVisible = true
+            ChatUI.quotedAttachmentFactoryManager.createQuotedView(message, binding.attachmentContainer)
         } else {
-            when (attachment.type) {
-                ModelType.attach_file -> showFileTypeLogo(attachment.mimeType)
-                ModelType.attach_image -> showAttachmentThumb(attachment.imagePreviewUrl)
-                ModelType.attach_giphy,
-                ModelType.attach_video,
-                -> showAttachmentThumb(attachment.thumbUrl)
-                else -> showAttachmentThumb(attachment.image)
-            }
+            binding.attachmentContainer.isVisible = false
```

Comment:
In the previous condition, we set the `logoContainer` visibility to false and here we change the attachment container - any reason why this change?

Reply:
As mentioned down below, it is just a container where the attachment is populated.